Source/ProgressionSystemRuntime/Public/Data/PSTypes.h

```diff
@@ -60,4 +60,8 @@ struct FPSRowData : public FTableRowBase
 	/** Defines if level is locked or not */
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
 	bool IsLevelLocked = true;
+
+	/** Defines the star animation for each character */
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
+	TObjectPtr<class UCurveTable> HideStarsAnimation;
```

Comment:
inline init the property with `= nullptr;`

Reply:
done as part of https://github.com/h4rdmol/ProgressionSystem/pull/12/commits/0dc42fbdf6ceb67efd71cbafead115ade2be8ad1